easybuild/easyconfigs/f/FLINT/FLINT-2.5.2-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'FLINT'
+version = '2.5.2'
+
+homepage = 'http://www.%(namelower)slib.org'
+
+description = """FLINT (Fast Library for Number Theory) is a C library in support of computations
+ in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs,
+ factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides
+ various low-level routines for fast arithmetic. FLINT is extensively documented and tested."""
+
+toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
+
+source_urls = ['http://www.%(namelower)slib.org']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['cbf1fe0034533c53c5c41761017065f85207a1b770483e98b2392315f6575e87']
+
+dependencies = [
+    ('GMP', '6.1.2'),
+    ('MPFR', '4.0.1'),
+]
+
+configopts = '--with-gmp=$EBROOTGMP --with-mpir=$EBROOTMPIR'
```

Comment:
Ahhh, this is wrong.
You have a dep for MPFR, not MPIR, so it should be
--with-mpfr=$EBROOTMPFR

The same for the other FLINT easyconfig

Reply:
@akesandgren I changed this, sorry for the hassle.